server/startup/migrations/v236.ts

```diff
@@ -0,0 +1,21 @@
+import { Migrations } from '../../../app/migrations/server';
+import { Settings } from '../../../app/models/server';
+
+Migrations.add({
+	version: 236,
+	up() {
+		Settings.removeById('Canned Responses');
```

Comment:
Wasn't this removed some time ago too? 👀

Reply:
not for me... 👀